index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a

```

Comment:
Makes sense. @plehegar do you, by any chance, remember why the "or immediately before the transport connection is closed" was added here / was meant to capture? Was this a clause for HTTP/1.0 and lack of keep-alive?